src/app/pages/repoDetails/repoDetails.component.ts

```diff
@@ -0,0 +1,50 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute}   from '@angular/router';
+import {RepoService} from "../../services/repo.service";
+
+@Component({
+  selector: 'repoDetails',
+  templateUrl: './RepoDetails.template.html',
+})
+
+export class RepoDetails implements OnInit {
+  repoDetails: RepoDetails;
+  loadingTable = false;
+  table = true;
```

Comment:
As I mentioned in template's comment, no need in `table` flag, can use `repoDetails`.